packages/core/src/components/forms/radioGroup.tsx

```diff
@@ -86,11 +93,21 @@ export class RadioGroup extends AbstractPureComponent<RadioGroupProps> {
     private autoGroupName = nextName();
 
     public render() {
-        const { label } = this.props;
+        const { label, options, className, children, name, onChange, ...htmlProps } = this.props;
+        const labelId = uniqueId("label");
         return (
-            <div className={classNames(Classes.RADIO_GROUP, this.props.className)}>
-                {label == null ? null : <label className={Classes.LABEL}>{label}</label>}
-                {Array.isArray(this.props.options) ? this.renderOptions() : this.renderChildren()}
+            <div
+                className={classNames(Classes.RADIO_GROUP, className)}
+                role="radiogroup"
+                aria-labelledby={label ? labelId : undefined}
```

Comment:
Is it problematic that we now can pass both `aria-labelledby` and `label` to this component? Take the following example:
```tsx
<label id="external-label">My Label</label>
<RadioGroup
    label={<label id="prop-node-label">Determine lunch</label>}
    aria-labelledby="external-label"
    ...
>
    <Radio label="Soup" value="one" />
    <Radio label="Salad" value="two" />
    <Radio label="Sandwich" value="three" />
</RadioGroup>
```

Which "label" should take precedence?

Furthermore, we can theoretically achieve the proper aria labeling by passing in a label manually.
```tsx
<RadioGroup
    aria-labelledby="my-label"
    ...
>
    <Label id="my-label">My Label</Label>
    <Radio label="Soup" value="one" />
    <Radio label="Salad" value="two" />
    <Radio label="Sandwich" value="three" />
</RadioGroup>
```

IMO, the functionality of the `label` prop itself seems a bit ambiguous for this case. Perhaps it would ultimately be better if `<RadioGroup>` did away with this prop and instead associated any `<Label>` components passed as children with the radio group.

Reply:
> Is it problematic that we now can pass both aria-labelledby and label to this component? 

No, this allows a consumer to pass a custom `aria-labelledby` if they wish to use an external label.

> Which "label" should take precedence?

In your example, `external-label` takes precedence because the consumer intentionally passed the `aria-labelledby` prop. which is exactly the way they should do it if they want an external label. The internal label no longer labels it, but that's exactly what they wanted when they passed `aria-labelledby` themselves.

Reply:
@ggdouglas looks like this PR has been forgotten about, can you take a look?